cmd/entc/entc.go

```diff
@@ -194,15 +194,27 @@ func createDir(target string) error {
 	if err := os.MkdirAll(target, os.ModePerm); err != nil {
 		return fmt.Errorf("creating schema directory: %w", err)
 	}
-	if target != defaultSchema {
-		return nil
-	}
-	if err := ioutil.WriteFile("ent/generate.go", []byte(genFile), 0644); err != nil {
+	if err := ioutil.WriteFile(target+"/../generate.go", []byte(getGenerateFileData(target)), 0644); err != nil {
```

Comment:
Use `filepath.Dir` and `filepath.Join` for assembling the file path (it won't work in Windows for example).

Reply:
Code updated in https://github.com/facebookincubator/ent/pull/585/commits/59dcd397fd020d5bbc111f6c26756f213a39dc78.